CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+<a name="App Functions SDK ChangeLog"></a>
+## App Functions SDK (in Python)
+[Github repository](https://github.com/edgexfoundry/app-functions-sdk-python)
+
+### Change Logs for EdgeX Dependencies
+- [go-mod-bootstrap](https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/CHANGELOG.md)
+- [go-mod-core-contracts](https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/CHANGELOG.md)
+- [go-mod-messaging](https://github.com/edgexfoundry/go-mod-messaging/blob/main/CHANGELOG.md)
+- [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry/blob/main/CHANGELOG.md) 
+- [go-mod-configuration](https://github.com/edgexfoundry/go-mod-configuration/blob/main/CHANGELOG.md) (indirect dependency)
+- [go-mod-secrets](https://github.com/edgexfoundry/go-mod-secrets/blob/main/CHANGELOG.md) (indirect dependency)

```

Comment:
```suggestion
```
Remove these lines, as this is a python based SDK and doesn't depend on go based modules.